src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -605,7 +636,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   }
 
   def workflowGcsPath(descriptor: WorkflowDescriptor): String = {
-    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, jesConf.executionBucket)
+    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, jesConf.executionBucket).stripSuffix("/")
```

Comment:
I had hard coded in some GCS urls for testing and I found out that this is not resilient to trailing slashes because I had included a trailing slash in my hard-coded value and a few things broke because of the trailing slash